App/Containers/TextilePhotos.js

```diff
@@ -15,14 +15,11 @@ import BottomDrawerList from '../SB/components/BottomDrawerList'
 class TextilePhotos extends React.PureComponent {
   constructor (props) {
     super(props)
-    this.state = {
-      showDrawer: false
-    }
   }
```

Comment:
just did a bunch of cleanup in this container. it is only ever called in the default thread now so lots of code removed